conformance/lang/values-types-and-variables/string.balt

```diff
@@ -0,0 +1,64 @@
+Test-case: output
+Description: Test valid string values
+Label: string
+
+public function main() returns () {
+    string a = "Ballerina";
+    io:println(a); // @output "Ballerina"
```

Comment:
Should be

```
// @output Ballerina
```

without quotes and similarly for other cases.

Reply:
`a.toBalString()` prints with quotes on the terminal. We need to use that output right?

Reply:
Yes, we decided to use the result of the toBalString as the output,  because toString() doesn't support nil.

Reply:
I don't agree we decided that. That means the ballerinai/println is not compatible with ballerina/println.

Reply:
This is going to be a problem when testing nil values and structural values. Because toString() is not good at capturing structures. This was discussed in the original proposal. Also, testing is internal to the compiler, Hence I don't think test output should be compatible with ballerina/io output. 

Other option is to use separate functions other than println.

Reply:
Resolving as per the conclusion to use `toString()`